src/cljs/main/broadfcui/page/workspace/data/utils.cljs

```diff
@@ -20,7 +20,13 @@
                        (let [entities (case entity-type
                                         "sample_set" (:samples attrs)
                                         "pair_set" (:pairs attrs)
-                                        "participant_set" (:participants attrs))]
+                                        "participant_set" (:participants attrs)
+                                        ;; below is for non FC model sets
+                                        ;; they will end in "_sets"
+                                        ;; we need to remove that (hence the subs at length-4)
+                                        ;; then make it plural by adding an s
+                                        ;; then make it a keyword
+                                        ((keyword (str (subs entity-type 0 (- (count entity-type) 4)) "s")) attrs))]
```

Comment:
Nit: Instead of the magic number `4`, I prefer to describe where the number comes from. In this case: `(count "_set")`